Settings labels & descriptions  + DB Migrations Setup

mattyg opened the pull request:
Resolves #166 
Resolves #109

**Overview**
- add fields label & description to Settings model
- display label & description in frontend settings form
- add dependency `umzug` to api for managing db migrations
- setup db migrations management system
- create db migration to update initial settings with a default label & description
- make umzug cli available as command `yarn workspace api migrator`

Note that db migrations should work with a fresh db, or with the expected database schema before this PR (i.e. they can be safely run on a production database).

kristoferlund reviewed (changes requested):
Looks good! 

Migration of old settings is not handled though. We need that to upgrade the current TEC installation.

mattyg commented:
> Migration of old settings is not handled though. We need that to upgrade the current TEC installation.

Good call.  I just went ahead and implemented #109 here:

**Overview**
- added dependency `umzug`
- setup umzug for db migrations management
- run all pending migrations on server launch
- create db migration 1 to insert default settings (or skip if setting already exists), excluding label & description
- create db migration 2 to update default settings with default label & description

kristoferlund reviewed (approved):
Looks great!!
